src/composables/useFormatDateTime.js

```diff
@@ -58,14 +58,14 @@ export function useFormatDateTime(timestamp = Date.now(), opts = {}) {
 	const date = computed(() => new Date(unref(timestamp)))
 
 	const formattedFullTime = computed(() => {
-		const formatter = new Intl.DateTimeFormat(getCanonicalLocale(), wrappedOptions.value.format)
+		const formatter = new Intl.DateTimeFormat(getLanguage(), wrappedOptions.value.format)
 		return formatter.format(date.value)
 	})
 
 	/** Time string formatted for main text */
 	const formattedTime = computed(() => {
 		if (wrappedOptions.value.relativeTime !== false) {
-			const formatter = new Intl.RelativeTimeFormat(getCanonicalLocale(), { numeric: 'auto', style: wrappedOptions.value.relativeTime })
+			const formatter = new Intl.RelativeTimeFormat(getLanguage(), { numeric: 'auto', style: wrappedOptions.value.relativeTime })
```

Comment:
This is ok as this only is used for relative time which should be independent from **locale** and only depend on **lanuage**